### PR TITLE
Fix the bug that the `dump_pep_pages` command was broken

### PR DESCRIPTION
**Pull request by mozillazg**

### CODE REVIEW

File: peps/tests/test_commands.py [hunk 1]
```diff
@@ -33,3 +38,20 @@ def test_image_generated(self):
         img = Image.objects.get(page__path='dev/peps/pep-3001/')
         soup = BeautifulSoup(img.page.content.raw, 'lxml')
         self.assertIn(settings.MEDIA_URL, soup.find('img')['src'])
+
+    @override_settings(PEP_REPO_PATH=FAKE_PEP_REPO)
+    def test_dump_pep_pages_work(self):
+        call_command('generate_pep_pages')
+        stdout = io.StringIO()
+        sys.stdout = stdout
```
Comment (berkerpeksag) [inline peps/tests/test_commands.py]: We can replace monkeypatching with the following snippet:

```py
stdout = io.StringIO()
call_command('dump_pep_pages', stdout=stdout)
output = stdout.getvalue()
```

File: peps/management/commands/dump_pep_pages.py [hunk 2]
```diff
@@ -4,14 +4,14 @@
 from pages.models import Page
 
 
-class Command(BaeeCommand):
+class Command(BaseCommand):
```
Comment (berkerpeksag) [inline peps/management/commands/dump_pep_pages.py:7]: Nice catch!

File: peps/tests/test_commands.py [hunk 3]
```diff
@@ -33,3 +38,20 @@ def test_image_generated(self):
...
+        try:
+            call_command('dump_pep_pages')
+        finally:
+            sys.stdout = sys.__stdout__
+        stdout.seek(0)
+        output = stdout.read()
+        stdout.close()
+
+        json.loads(output)
```
Comment (berkerpeksag) [inline peps/tests/test_commands.py]: Do we need to call `json.loads()` here?

File: peps/tests/test_commands.py [hunk 4]
```diff
@@ -33,3 +38,20 @@ def test_image_generated(self):
         img = Image.objects.get(page__path='dev/peps/pep-3001/')
         soup = BeautifulSoup(img.page.content.raw, 'lxml')
         self.assertIn(settings.MEDIA_URL, soup.find('img')['src'])
+
+    @override_settings(PEP_REPO_PATH=FAKE_PEP_REPO)
+    def test_dump_pep_pages_work(self):
```
Comment (berkerpeksag) [inline peps/tests/test_commands.py]: `_work` can be removed from the test name.

File: peps/tests/test_commands.py (hunk 3 above)
Comment (mozillazg) [reply peps/tests/test_commands.py]: No, it's not necessary.

File: peps/tests/test_commands.py (hunk 1 above)
Comment (mozillazg) [reply peps/tests/test_commands.py]: 👍

**Review by berkerpeksag — APPROVED** [review]

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**